integrations/tensorflow/test/python/iree_tfl_tests/east_text_detector_test.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import absl.testing
+import numpy
+import iree.tflite.support.test_util as test_util
```

Comment:
We had this file in the wrong place and i just pushed a fix last night. You want this to be now:

`from . import test_util`